lib/matplotlib/artist.py

```diff
@@ -396,8 +396,8 @@ def _default_contains(self, mouseevent, figure=None):
                 return inside, info
             # subclass-specific implementation follows
 
-        The *figure* kwarg is provided for the implementation of
-        `Figure.contains`.
+        The *canvas* kwarg is provided for the implementation of
```

Comment:
But the kwarg is `figure`, so why change?

Reply:
probably because I was confused...